pom.xml

```diff
@@ -25,9 +25,10 @@
   </scm>
 
   <properties>
-    <maven.compiler.source>1.6</maven.compiler.source>
-    <maven.compiler.target>1.6</maven.compiler.target>
+    <maven.compiler.source>1.7</maven.compiler.source>
```

Comment:
I don't think it's necessary to set the compiler or target version to 1.7, just that the runtime JRE be 1.7.

Am I mistaken?